doc/releases/changelog-dev.md

```diff
@@ -291,6 +291,9 @@
 
 <h3>Bug fixes 🐛</h3>
 
+* Using `@` with legacy Hamiltonian instances now properly de-queues the previously existing operations.
```

Comment:
What happens in the new-op-arithmetic setting?

Reply:
Things are properly de-queued.